operations/observability/mixins/workspace/rules/components/workspaces/alerts.libsonnet

```diff
@@ -43,6 +43,20 @@
               ) without(phase) > 5
             |||,
           },
+          {
+            alert: 'GitpodWorkspaceHighFailureRate',
```

Comment:
hm. this alert seems to be missing a `for` statement. I think it might default to 1min or something in that case?

Reply:
we are calculating rate for five minutes itself so we don't need a for statement.